api/errors.py

```diff
@@ -1,13 +1,29 @@
-MISSING_TOKEN = 'Token is missing!'
-EXPIRED_TOKEN = 'Your token expired! Please refresh it!'
-INVALID_TOKEN = 'Token is invalid!'
+from http import HTTPStatus
 
 
-class TokenError(Exception):
-    def __init__(self, message):
+class ApiKeyError(Exception):
+    def __init__(self, message, status):
         super().__init__()
         self.message = message
+        self.status = status
 
     @property
     def json(self):
-        return {'message': self.message}
+        return {
+            'status': self.status,
+            'message': self.message,
+        }
+
+
+class AuthorizationError(ApiKeyError):
+    def __init__(self):
+        super().__init__('Invalid api key', HTTPStatus.BAD_REQUEST)
+
+
+class RequestError(ApiKeyError):
+    def __init__(self, message):
+        super().__init__(message, status=HTTPStatus.BAD_REQUEST)
+
+
+# class MissingApiKeyError(ApiKeyError):
```

Comment:
I think it is better to delete these comments

Reply:
https://github.com/mstoro/nasa_api/pull/1/commits/72d016018bdc3cc1c5284981c061931570064aea